src/utils/managing.officer.corporate.ts

```diff
@@ -72,6 +75,14 @@ export const getManagingOfficerCorporateById = (req: Request, res: Response, nex
       templateOptions[StartDateKey] = startDate;
     }
 
+    // Start: Redis removal work - Add extra template options if Redis Remove flag is true - needs refacotoring into a common method
+    const isRegistration: boolean = req.path.startsWith(config.LANDING_URL);
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL) && isRegistration) {
```

Comment:
This is encapsulated in a new method (`addActiveSubmissionBasePathToTemplateData()`) in `/src/utils/template.data.ts`

Reply:
See above.